client/task-list/index.js

```diff
@@ -406,6 +407,12 @@ export default compose(
 		const activePlugins = getActivePlugins();
 		const installedPlugins = getInstalledPlugins();
 
+		const wcPaySettings =
+			getOption( 'woocommerce_woocommerce_payments_settings' ) || false;
+
+		taskListPayments.wcPayCompleted =
+			wcPaySettings && wcPaySettings.enabled === 'yes';
```

Comment:
I think we'll need to also check if this is configured.  See `wcPayIsConnected` for more details.